CHANGES.rst

```diff
@@ -1,15 +1,51 @@
 Changes
 =======
 
-Version 0.9.2 (UNRELEASED)
+Version 0.9.2 (2023-12-13)
 --------------------------
 
+- Users:
+    - Adds form to generate the launcher URL of any user-provided analysis, together with the markdown snippet for the corresponding Launch-on-REANA badge.
+    - Adds option to delete all the runs of a workflow from the web interface.
+    - Changes ``validate`` command to show detailed errors when the specification file is not a valid YAML file.
+    - Changes CVMFS support to allow users to automatically mount any available repository.
+    - Changes the Launch-on-REANA page to improve how workflow parameters are shown by displaying them inside a table.
+    - Changes the workflow deletion endpoint to return a different and more appropriate message when deleting all the runs of a workflow.
+    - Changes validation of REANA specification to make the ``environment`` property mandatory for the steps of serial workflows.
+    - Changes validation of REANA specification to raise a warning for unexpected properties for the steps of serial workflows.
+    - Changes validation of REANA specification to report improved validation warnings which also indicate where unexpected properties are located in the file.
+    - Changes workflow restarts to allow for more than nine restarts of the same workflows.
```

Comment:
BTW for the nine restarts news item, the better leading word would be "Fixes...", but since the components where this was implemented are already released, let's keep "Changes..." here.